### ISSUE TITLE
Bundlephobia fails to analize bundle due to missing dependencies 🤔

### ISSUE THREAD

**Issue by emmenko** [tdeekens/flopflip#609]
https://bundlephobia.com/result?p=@flopflip/react

<img width="1359" alt="image" src="https://user-images.githubusercontent.com/1110551/53694927-625abf00-3db5-11e9-8f47-81d32ac856fd.png">

Any idea what's wrong?

**Comment by tdeekens** [comment]
No. Will look into it a bit.

**Comment by tdeekens** [comment]
Ah good old babel runtime. The transform is used so that not every use of the babel helpers adds them to every file that requires it. The `@babel/runtime` however is missing as a production dependency. Which to my understanding is not a probblem in itself as any consumer should have it (maybe I should make it a peer dep) as the runtime will be compiled into an apps build in the end.

Anyway, I think Bundlephobia doesn't "just look" the what in dist but actually tries to optimize it again (to gzip etc). There it might notice that something is fishy from its view.

Long story short: I added `@babel/runtime` here as a dep as it won't harm. I also took the change th re-work the badges a bit. I don't want to force publish a new version but it should solve itself in the next release.

Will keep this open until then.

**Comment by emmenko** [comment]
Awesome! 👌

**Comment by tdeekens** [comment]
Hm, bundlephobia doesn't seem to handle monorepos all too well yet (also kind of of logical why). It seems like the `@babel/runtime` has to be a depdenency on any package (not the workspace root) as it installs all dependencies from there for the package being checked.

**Comment by emmenko** [comment]
I thought it just uses the npm package. Are you sure it looks for the github repo? 🤔

**Comment by tdeekens** [comment]
It uses the npm package and whatever you bundled. However it also wants to include adjacient dependencies included in your code. So like `@babel/runtime` or `corejs` which ultimatively end up with your code at some point. For that Bundlephobia actually installs the `dependencies` and bundles again.

This is now a working `@flopflip/splitio-adapter` metric: https://bundlephobia.com/result?p=@flopflip/splitio-adapter@1.3.8

<img width="1686" alt="cleanshot 2019-03-05 at 13 46 48 2x" src="https://user-images.githubusercontent.com/1877073/53806383-235f7180-3f4d-11e9-942e-33a2e11e2b50.png">

**Comment by tdeekens** [comment]
@flopflip/react-broadcast now also works. https://bundlephobia.com/result?p=@flopflip/react-broadcast@8.0.1. 

Readme badges now also do their jobs.

<img width="942" alt="cleanshot 2019-03-05 at 13 47 53 2x" src="https://user-images.githubusercontent.com/1877073/53806448-5275e300-3f4d-11e9-991d-9af5f5463d95.png">

I'll close this and take potential improvements from here (e.g. getting rid of lodash again).

**Comment by emmenko** [comment]
🎉
